packages/mutt.rb

```diff
@@ -3,17 +3,17 @@
 class Mutt < Package
   description 'Mutt is a small but very powerful text-based mail client for Unix operating systems.'
   homepage 'http://mutt.org/'
-  version '1.8.2'
-  source_url 'ftp://ftp.mutt.org/pub/mutt/mutt-1.8.2.tar.gz'
-  source_sha256 '1d057bf1b565f2c38ee50c9a661654cbbe4165f98e25bfa361ebbd707d96f235'
+  version '1.8.3'
+  source_url 'ftp://ftp.mutt.org/pub/mutt/mutt-1.8.3.tar.gz'
+  source_sha256 '9b81746d67ffeca5ea44f60893b70dc93c86d4bc10187d4dd360185e4d42ed42'
 
   def self.build
-    system 'mkdir /usr/local/mail'
```

Comment:
I didn't see that error during my testing but I put it back and pushed another commit.